src/plugins/visualizations/public/visualize_app/components/visualize_listing.tsx

```diff
@@ -64,6 +65,10 @@ export const VisualizeListing = () => {
     }
   }, [history, pathname]);
 
+  useCustomEventContext({
```

Comment:
It probably doesn't but just to make sure - this might be called very often even if the user stays on the current page. It shouldn't trigger any requests on its own.